
时间：2016-10-08 08:47:10,621 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(D:\Pro\x.shop\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：33563 
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: TCP Provider, error: 0 - 等待的操作过时。) ---> System.ComponentModel.Win32Exception (0x80004005): 等待的操作过时。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   在 System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   在 System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   在 System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   在 System.Data.Linq.Table`1.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   在 System.Linq.Queryable.SingleOrDefault[TSource](IQueryable`1 source, Expression`1 predicate)
   在 X.App.Apis.mgr.login.Execute() 位置 D:\Pro\x.shop\src\Web\O2O\X.App\Apis\mgr\login.cs:行号 39
   在 X.Web.Apis.Api.GetResponse() 位置 D:\Pro\x.shop\src\Web\X.Web\Apis\Api.cs:行号 30
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 D:\Pro\x.shop\src\Web\X.Web\XApp.cs:行号 31
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:258,State:0,Class:20

时间：2016-10-08 08:48:08,536 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.shop\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(T验证码不正确)","code":"T验证码不正确"} 
时间：121 

时间：2016-10-08 08:48:14,530 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.shop\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"未知错误代码(用户名或密码不正确)","code":"用户名或密码不正确"} 
时间：6115 

时间：2016-10-08 14:11:40,063 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.shop\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"缺少参数 商品类型","code":"0x0003"} 
时间：69950 

时间：2016-10-08 14:14:47,293 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(D:\Pro\x.shop\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"缺少参数 商品类型","code":"0x0003"} 
时间：257180 
